doc/classes/Object.xml

```diff
@@ -1040,6 +1040,7 @@
 		</constant>
 		<constant name="CONNECT_PERSIST" value="2" enum="ConnectFlags">
 			Persisting connections are stored when the object is serialized (such as when using [method PackedScene.pack]). In the editor, connections created through the Node dock are always persisting.
+			[b]Note:[/b] Connections to lambda functions (i.e. when the function code is embedded in the [method connect] call) cannot be made persistent.
```

Comment:
Okay so, this ambiguity is why I personally adamantly avoid both "e.g." and "i.e.": Because of what "i.e." literally stands for, this note strongly implies that "lambda" functions are when code is embedded in `connect()`, and that's really the only exception to `CONNECT_PERSIST`.

We know both statements aren't true, or at least it's quite misleading, so no point explaining it. However, I must note that we're working with "Callables" here, not just "functions".
In fact, what could be worth testing for this purpose is one _fire-sure way_ to [determine whether a Callable is a lambda or not](https://docs.godotengine.org/en/4.5/classes/class_callable.html#class-callable-method-is-custom).

No suggestion yet, it's late at night.